proposals/lip-0053.md

```diff
@@ -305,6 +305,10 @@ def verifyPartnerChainOutboxRoot(ccu: CCU) -> None:
 ```python
 def verifyCommon(ccu: CCU) -> None:
     validateObjectSchema(crossChainUpdateTransactionParams, ccu.params)
+
+    if ccu.params.sendingChainID == OWN_CHAIN_ID:
```

Comment:
Maybe there was a misunderstanding in our communication: I don't think this one is necessary because we have already the check `if chainAccount(ccu.params.sendingChainID) does not exist` (introduced in this PR). But it also doesn't hurt to have it.

Reply:
yeah, I realized this afterwards but then I thought that this would be a quick check that would immediately fail without needing to read from the state.